vac/raw/sds.md

```diff
@@ -69,12 +69,17 @@ Messages MUST adhere to the following meta structure:
 ```protobuf
 syntax = "proto3";
 
+message HistoryEntry {
+  string message_id = 1; // Unique identifier of the SDS message, as defined in `Message`
+  optional bytes retrieval_hint = 2; // Optional information to help remote parties retrieve this SDS message; For example, A Waku deterministic message hash or routing payload hash
+}
+
 message Message {
   // 1 Reserved for sender/participant id
   string message_id = 2;          // Unique identifier of the message
   string channel_id = 3;          // Identifier of the channel to which the message belongs
```

Comment:
While we're here, should we review the field types for the rest of the `Message`? The choice of `string` for `message_id` and `channel_id` was heavily influenced by similar protocols and current Status code, but might not be the most practical from implementation perspective: I can imagine `bytes` perhaps being more reasonable for `message_id` and `int32` for `channel_id`.

Any opinions here? cc @shash256

Reply:
Opened a separate issue for this: https://github.com/vacp2p/rfc-index/issues/137